src/app/api/fetchData.js

```diff
@@ -0,0 +1,13 @@
+import { countriesAxios } from "@/countriesAxios"
+
+const fetchData = async (endpoint, options = {filters: ''}) => {
+    const filtersValues = String(options.filters)
```

Comment:
Rozumiem, że tutaj konwertujesz Array na String. A może, żeby znać kontekst tej zmiennej (Array) od razu zrobić to tak:
```suggestion
    const filtersValues = options.filters.join(',')
```
?